test/test_output.py

```diff
@@ -18,7 +18,6 @@
 
 from fontTools.ttLib import TTFont
 
-
```

Comment:
Oh Thanks for pointing the PEP8 style guide. It's worth reading and I looked though it. I made some style changes in the codebase based on its recommendation.